alert-database/src/main/java/com/synopsys/integration/alert/database/job/JobConfigurationModelFieldExtractorUtils.java

```diff
@@ -66,10 +81,12 @@ public static DistributionJobModel convertToDistributionJobModel(UUID jobId, Map
                                                   .policyFilterPolicyNames(extractFieldValues("blackduck.policy.notification.filter", configuredFieldsMap))
                                                   .vulnerabilityFilterSeverityNames(extractFieldValues("blackduck.vulnerability.notification.filter", configuredFieldsMap));
 
-        List<BlackDuckProjectDetailsModel> blackDuckProjectDetails = extractFieldValues("channel.common.configured.project", configuredFieldsMap)
-                                                                         .stream()
-                                                                         .map(projectName -> new BlackDuckProjectDetailsModel(projectName, projectName))
-                                                                         .collect(Collectors.toList());
+        List<BlackDuckProjectDetailsModel> blackDuckProjectDetails = Optional.ofNullable(projectFilterDetails)
```

Comment:
Isn't this an example of a way to **not** use an optional? Instead of being passed a null list, we should be passing an empty list which we then check for content and modify/use as needed.